src/JSystem/JStage/JSGActor.cpp

```diff
@@ -1 +1,78 @@
-#include <JSystem/JStage/JSGActor.hpp>
+#include "JSystem/JStage/JSGActor.hpp"
+#include "dolphin/types.h"
+
+/* 8026DBBC-8026DC1C       .text __dt__Q26JStage6TActorFv */
```

Comment:
purge these comments, they are not relevant for sms (offsets would be different)